projects/personality_captions/transresnet/modules.py

```diff
@@ -399,9 +399,9 @@ def choose_best_caption(
         for img_index in range(len(context_encoded)):
             context_encoding = context_encoded[img_index : img_index + 1, :]
             scores = torch.mm(
-                candidates_encoded[img_index]
+                candidates_encoded[img_index].to(context_encoding.device)
```

Comment:
Nit: I believe you can just do `.to(context_encoding`

Reply:
rly? that's new to me.

Reply:
```python
In [1]: import torch

In [2]: torch.__version__
Out[2]: '1.4.0'

In [3]: a = torch.LongTensor([1, 2, 3]).cuda()

In [4]: a
Out[4]: tensor([1, 2, 3], device='cuda:0')

In [5]: b = torch.LongTensor([4, 5, 6]).to(a)

In [6]: b
Out[6]: tensor([4, 5, 6], device='cuda:0')

```

Reply:
Yes, changed